README.md

```diff
@@ -26,53 +49,28 @@ imagelab.find_issues()
 imagelab.report()
 ```
 
-CleanVision diagnoses many types of issues, but you can also check for only specific issues:
-
+3. This step is optional. You can also specify specific issues instead of checking for all issues.
+The **Issue Key** column in the table above specifies the name for each type of issue in the CleanVision code. Use these string names to specify which types of issues should be checked.
 ```python
-issue_types = {"light": {}, "blurry": {}}
+issue_types = {"dark": {}, "blurry": {}}
 
-imagelab.find_issues(issue_types)
+imagelab.find_issues(issue_types=issue_types)
 
 # Produce a report with only the specified issue_types
-imagelab.report(issue_types.keys())
+imagelab.report(issue_types=issue_types)
 ```
 
-More on how to get started with CleanVision:
-
-- [Example script](https://github.com/cleanlab/cleanvision/blob/main/examples/run.py)
-- [Jupyter notebook tutorial](https://github.com/cleanlab/cleanvision/blob/main/examples/demo.ipynb)
+## More on how to use CleanVision
+- [Tutorial notebook](https://github.com/cleanlab/cleanvision/blob/main/examples/demo.ipynb)
 - [Documentation](https://cleanvision.readthedocs.io/)
+- [Example script](https://github.com/cleanlab/cleanvision/blob/main/examples/run.py)
 
```

Comment:
link to the cleanvision-examples repo here too?
https://github.com/cleanlab/cleanvision-examples